logback-android/src/main/java/ch/qos/logback/classic/util/LogbackMDCAdapter.java

```diff
@@ -195,4 +198,24 @@ public void setContextMap(Map<String, String> contextMap) {
     // the newMap replaces the old one for serialisation's sake
     copyOnThreadLocal.set(newMap);
   }
+
+  @Override
+  public void pushByKey(String key, String value) {
```

Comment:
Why are the changes in this file are necessary for this PR?

Reply:
It looks like its because of changes to the MDCAdapter interface in slf4fj 2.0: https://www.slf4j.org/api/org/slf4j/spi/MDCAdapter.html